src/js/app.js

```diff
@@ -1,3 +1,23 @@
 import '../scss/main.scss';
 
 console.log("Hello, Webpack!");
+
+//función para capturar el evento cuando el usuario hace scroll y asignarle la clase sticky
+function sticky() {
+    var header = document.getElementsByClassName('header')[0],
+        top = 0;
+
+    function scroll(event) {
+        var s = document['documentElement' || 'body'].scrollTop;
+        if (s > top) header.classList.add('header__sticky');
+        else header.classList.remove('header__sticky');
```

Comment:
Please use the curly braces to separate the `if` and `else` statements.
Like this
```suggestion
        if (s > top) {
          header.classList.add('header__sticky');
        } else {
          header.classList.remove('header__sticky');
        }
```